py/src/braintrust/framework.py

```diff
@@ -280,9 +280,12 @@ def Eval(
             loop = None
 
         async def run_to_completion():
-            with init_experiment(evaluator.project_name, evaluator.experiment_name, evaluator.metadata) as experiment:
+            experiment = init_experiment(evaluator.project_name, evaluator.experiment_name, evaluator.metadata)
```

Comment:
are we removing the context manager altogether? What happens if someone uses it?

Reply:
It's not removed. Just a no-op now.

Reply:
Ah cool